app/jobs/delete_events_job.rb

```diff
@@ -0,0 +1,12 @@
+class DeleteEventsJob < ActiveJob::Base
+  def perform(dump_type:, older_than:)
+    start = Event.arel_table[:start]
+    ids = Event
+          .joins(dump: :dump_type)
+          .where('dump_types.constant': dump_type)
+          .where(start.lt(older_than))
```

Comment:
Took me a minute to grok that older date times are less than (<) newer date times. Seconds since the Unix Epoch rather that seconds since the current time. TIL.